tests/mtx/device.type.test.ts

```diff
@@ -116,8 +440,9 @@ describe(`MTX device types (${deviceTypes.length})`, () => {
             let buffer = new CommandBinaryBuffer(hex);
             const deviceType = buffer.getDeviceType();
 
-            expect(deviceType.type).toBe(type);
-            expect(deviceType.revision).toBe(revision);
+            expect(deviceType).toBeDefined();
+            expect(deviceType?.type).toBe(type);
```

Comment:
seems `deviceType.type` should be always present

Reply:
fixed